client/src/components/SimpleDataField.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+const TextareaContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
```

Comment:
you can use the shorthand `flex-flow` here

Reply:
Of course!